presentation/src/main/java/com/sopt/presentation/appointmentCreate/appointmentCreateInfo/AppointmentCreateInfoRoute.kt

```diff
@@ -87,11 +122,16 @@ fun AppointmentCreateInfoScreen(
     groupId: Long,
     onButtonClick: (Long, String, String, Int) -> Unit,
     onBackButtonClick: () -> Unit,
-    categories: List<String>
+    categories: List<String>,
+    snackBarHostState: SnackbarHostState,
+    snackBarVisible: MutableState<Boolean>,
+    showSnackBar: () -> Unit
 ) {
     var appointmentName by remember { mutableStateOf("") }
     var appointmentCategory by remember { mutableStateOf("") }
     var appointmentDuration by remember { mutableStateOf("") }
+    var hasInput by remember { mutableStateOf(false) }
+    val isOnlySpace = hasInput && appointmentName.isNotEmpty() && appointmentName.trim().isEmpty()
```

Comment:
```suggestion
    val isOnlySpace = hasInput && appointmentName.isBlank
```
P1: isBlank() 는 isEmpty()와 달리 공백도 true 처리해줘서 이렇게 써도 되지 않나요??